media/css/firefox/family-nav.less

```diff
@@ -281,7 +281,7 @@
                 opacity: 0.8;
                 height: 77px;
                 line-height: 77px;
-                max-width: 125px; // uhhh, seems to work?
+                //max-width: 145px;
```

Comment:
I think we should keep this restriction in place, and should set it to `129px`. At tablet width, the `#desktop-subnav` is 556px wide, which, divided by 4 (the max elements now possible in the subnav) is 139. Each `<a>` has a left and right padding of 5px, hence 129.

If this restriction isn't in place, text will not wrap to two lines, which means an `<li>` could get pushed underneath and out of view for some locales.